Dockerfile-devshell

```diff
@@ -0,0 +1,45 @@
+# Uwazi Development Container (not for production!)
+
+# https://getfedora.org
+FROM fedora:30
+
+# Initially built by Michael Vorburger.ch at Google for Huridocs, later maintained by the Uwazi open source community
+LABEL maintainer="Michael Vorburger.ch <vorburger@google.com>"
+
+# Update base image (the public one is typically outdated and contains already patched security vulnerabilities)
+RUN dnf update -y
+
+# https://docs.mongodb.com/v3.4/tutorial/install-mongodb-on-red-hat/
+COPY mongodb-org-3.4.repo /etc/yum.repos.d/mongodb-org-3.4.repo
+
+# Install utility packages generally useful in any devshell
+RUN dnf install -y git findutils less which \
+
+# Install system packages specifically required by Uwazi
+    libpng12-devel libpng-devel mongodb-org-shell mongodb-org-tools poppler
```

Comment:
How do you control the poppler version? I think it needs to be 0.26 specifically.

Reply:
> How do you control the poppler version? I think it needs to be 0.26 specifically.

@bdittes We can't control the version of an (RPM or DEB) OS package .. e.g. on the `fedora:30` used as the base image for this container, the `poppler-utils` package comes with `pdftotext -v` => `pdftotext version 0.73.0` ... (see the _Packaged Versions_ on https://poppler.freedesktop.org).

The `node:8-jessie` used as based image over in https://github.com/fititnt/uwazi-docker/blob/master/Dockerfile indeed includes the much older `pdftotext version 0.26.5`.

@konzz @txau (or @fititnt ?) are there any known incompatibilities in up-to-date Poppler versions like 0.73.0 which make them totally un-usable for Uwazi? I guess it would be possible to try to rebuild an ancient, from source (in another `Dockerfile..`), but... unless there are very strong reasons, why would we?

PS: I noticed while checking these versions that `poppler` was actually the wrong :smiling_imp:  package by mistake, it's `poppler-utils` - have just fixed that in a new commit.

Reply:
BTW the https://github.com/huridocs/uwazi/blob/development/.circleci/config.yml in lines 29 both 89 also just does `sudo apt-get install poppler-utils`, so depending on what version of Debian CircleCI runs, which does not seem to be fixed and which they could change any time (I've actually run into similar problems on TravisCI in another project in the past), you get a Poppler version you can't control either in CI ...

Reply:
I use locally the 0.74.0 and have had no problems at all.

I think we should change the dependency number in the README.